src/arrayMethodReduce.js

```diff
@@ -3,9 +3,19 @@
 /**
  * Implement method Reduce
  */
+
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function(callback, initialValue = this[0]) {
+    let accumulator = initialValue;
+    let i;
+
+    (arguments.length === 1) ? i = 1 : i = 0;
```

Comment:
```suggestion
    let i = arguments.length === 1 ? 1 : 0;
```
говорят, в тернарниках пишут так)